indexer/packages/postgres/src/db/migrations/migration_files/20240208161948_add_orders_clobpairid_subaccountid_index.ts

```diff
@@ -0,0 +1,18 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  // eslint-disable-next-line @typescript-eslint/quotes
+  await knex.raw(`
+    CREATE INDEX CONCURRENTLY IF NOT EXISTS orders_clobpairid_subaccountid_index ON orders("clobPairId", "subaccountId");
```

Comment:
nit: we are inconsistent with our indices where in some we include `id` (e.g. `perpetual_positions_subaccount_perpetual_created_at_index`) and in others we use the exact same case and use quotes (e.g. `fills_clobPairId_createdAtHeight_partial`).

Reply:
I will go with camel case for consistency